connectivity_summary.py

```diff
@@ -352,10 +352,15 @@ def path(self):
         if self._site_path is None:
             date, slice, site = self.expt_id.split('-')
             root = os.path.dirname(self.entry.file)
+            if len(date.split('_')) > 1:
+                date, specimen = date.split('_')
+                specimen = '_' + specimen
+            else: 
+                specimen = "_000"
```

Comment:
yep! didn't know you could do that :) do you want to just switch that and then merge it?